.changelog/162.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
```

Comment:
Debating whether this should be considered a breaking change or not (and the corresponding HTTPRoute change from earlier today). Thoughts?

We marked the initial support for `ReferencePolicy` as a breaking change [here](https://github.com/hashicorp/consul-api-gateway/blob/main/.changelog/143.txt), so it's finding the balance of which thing(s) are actually breaking.

Reply:
I think I'm comfortable with keeping just the validation check as the breaking change, but I'll PR something to make that entry a little more explicit about the breaking behavior - the watch and reconciliation trigger are only "breaking" as a byproduct of the validation check now failing.

Reply:
https://github.com/hashicorp/consul-api-gateway/pull/164/